src/dmtest/vdo/dedupe_tests.py

```diff
@@ -0,0 +1,91 @@
+from dmtest.assertions import assert_raises, assert_equal, assert_near
```

Comment:
assert_raises, assert_equal don't appear to be used

Reply:
fixed